src/js/features/beacon/beaconConfig.js

```diff
@@ -0,0 +1,36 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
this file doesn't follow the naming scheme for other public features

Reply:
It sure did when I wrote the code! Pretty much all of the beacon redux is just  modelled after what was in the codebase at the time. 

The redux code has changed quite a bit since then, beyond just the naming scheme. One of the tasks I've left for myself is to align the beacon redux with the current redux elsewhere. I'm not convinced this needs to be done immediately.